src/seedsigner/helpers/qr.py

```diff
@@ -48,6 +48,6 @@ def qrimage_io(self, data, width=240, height=240, border=3, background_color="80
         # if qrencode fails, fall back to only encoder
         if rv != 0:
             return self.qrimage(data,width,height,border)
-        img = Image.open("/tmp/qrcode.png").resize((width,height), Image.NEAREST).convert("RGBA")
+        img = Image.open("/tmp/qrcode.png").resize((width,height), Image.Resampling.NEAREST).convert("RGBA")
```

Comment:
I assume this removes a future deprecation warning.

Reply:
Yes, I forgot to note that in the description. The deprecation warning only appears (afaict) when running the test suite, but since I was trying to get everything in there neatly tied up, it was too glaring not to fix, too!